pkgs/development/ocaml-modules/chacha/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildDunePackage
+, fetchFromGitHub
+, ocaml
+
+, alcotest
+, cstruct
+, mirage-crypto
+}:
+
+buildDunePackage rec {
+  pname = "chacha";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
```

Comment:
Use `fetchurl` and <https://github.com/abeaumont/ocaml-chacha/releases/download/1.0.0/chacha-1.0.0.tbz> instead. This is the source tarball used by OPAM (and the opam hash will match) usually these release tarballs have some preprocessing done, so they are not identical to the source tarball generated by GitHub.